src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php

```diff
@@ -56,39 +56,39 @@ class CardSchemeValidator extends ConstraintValidator
             '/^65[0-9]{14}$/',
             '/^622(12[6-9]|1[3-9][0-9]|[2-8][0-9][0-9]|91[0-9]|92[0-5])[0-9]{10}$/',
         ),
-        /**
+        /*
          * InstaPayment cards begin with 637 through 639 and have 16 digits.
          */
         'INSTAPAYMENT' => array(
             '/^63[7-9][0-9]{13}$/',
         ),
-        /**
+        /*
          * JCB cards beginning with 2131 or 1800 have 15 digits.
          * JCB cards beginning with 35 have 16 digits.
          */
         'JCB' => array(
             '/^(?:2131|1800|35[0-9]{3})[0-9]{11}$/',
         ),
-        /**
+        /*
          * Laser cards begin with either 6304, 6706, 6709 or 6771 and have between 16 and 19 digits.
          */
         'LASER' => array(
             '/^(6304|670[69]|6771)[0-9]{12,15}$/',
         ),
-        /**
+        /*
          * Maestro cards begin with either 5018, 5020, 5038, 5893, 6304, 6759, 6761, 6762, 6763 or 0604
          * They have between 12 and 19 digits.
          */
         'MAESTRO' => array(
             '/^(5018|5020|5038|6304|6759|6761|676[23]|0604)[0-9]{8,15}$/',
         ),
-        /**
+        /*
          * All MasterCard numbers start with the numbers 51 through 55. All have 16 digits.
          */
         'MASTERCARD' => array(
             '/^5[1-5][0-9]{14}$/',
         ),
-        /**
+        /*
          * All Visa card numbers start with a 4. New cards have 16 digits. Old cards have 13.
          */
         'VISA' => array(
```

Comment:
To be honest, I think that 3 lines of comment for 1 or 2 lines of description is overkill. In my opinion the `// single line comment` would be sufficient.